[2025-07-26T09:10:47.624+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-26T09:10:47.678+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_postgres_test.load_user_dim_table scheduled__2025-07-26T08:00:00+00:00 [queued]>
[2025-07-26T09:10:47.708+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_postgres_test.load_user_dim_table scheduled__2025-07-26T08:00:00+00:00 [queued]>
[2025-07-26T09:10:47.727+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-07-26T09:10:47.806+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_user_dim_table> on 2025-07-26 08:00:00+00:00
[2025-07-26T09:10:47.829+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_postgres_test', 'load_user_dim_table', 'scheduled__2025-07-26T08:00:00+00:00', '--job-id', '146', '--raw', '--subdir', 'DAGS_FOLDER/my_dags.py', '--cfg-path', '/tmp/tmpighsbuv_']
[2025-07-26T09:10:47.841+0000] {standard_task_runner.py:105} INFO - Job 146: Subtask load_user_dim_table
[2025-07-26T09:10:47.837+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=265) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-26T09:10:47.844+0000] {standard_task_runner.py:72} INFO - Started process 278 to run task
[2025-07-26T09:10:48.073+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_postgres_test.load_user_dim_table scheduled__2025-07-26T08:00:00+00:00 [running]> on host 72bd964f8e11
[2025-07-26T09:10:48.907+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_postgres_test' AIRFLOW_CTX_TASK_ID='load_user_dim_table' AIRFLOW_CTX_EXECUTION_DATE='2025-07-26T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-26T08:00:00+00:00'
[2025-07-26T09:10:48.911+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-07-26T09:10:48.974+0000] {my_dags.py:297} INFO - Loading user dimension table
[2025-07-26T09:10:48.980+0000] {my_dags.py:317} INFO - Inserted 0 records into users table
[2025-07-26T09:10:48.983+0000] {my_dags.py:320} INFO - User dimension table loaded successfully
[2025-07-26T09:10:48.985+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-26T09:10:49.025+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-07-26T09:10:49.048+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_postgres_test, task_id=load_user_dim_table, run_id=scheduled__2025-07-26T08:00:00+00:00, execution_date=20250726T080000, start_date=20250726T091047, end_date=20250726T091049
[2025-07-26T09:10:49.144+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-26T09:10:49.216+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
